src/modules/contact.js

```diff
@@ -0,0 +1,49 @@
+function setSwap(id) {
+  const actBtn = document.querySelector('.tab.active');
+  if (actBtn) actBtn.classList.remove('active');
+
+  const homeBtn = document.getElementById(id);
+  homeBtn.classList.add('active');
+}
+
+function addSection(name, data) {
+  const section = document.createElement('section');
+  section.className = 'section font-mono mt-5 capitalize mx-5';
+
+  const title = document.createElement('h2');
+  title.className = ' font-bold text-4xl';
+  title.textContent = name;
+  section.appendChild(title);
+
+  data.forEach((d) => {
+    const info = document.createElement('p');
+    info.className = '';
+    info.textContent = d;
+    section.appendChild(info);
+  });
+
+  return section;
+}
+
+function contact() {
```

Comment:
## Suggestion 
- [ ] Do you care to implement this `contact` function declaration using `ES6`?
```suggestion
const contact = () => {
```